application/world/services/auto_bible_generator.py

```
@@ -866,6 +924,38 @@ async def _call_llm_and_parse(self, system_prompt: str, user_prompt: str) -> Dic
+        self,
+        system_prompt: str,
+        user_prompt: str,
+        max_retries: int = 3
+    ) -> Dict[str, Any]:
+        """带重试的LLM调用 - 增强JSON输出稳定性"""
+        last_error = None
+
+        for attempt in range(max_retries):
+            try:
+                if attempt == 0:
+                    # 第一次尝试，使用标准prompt
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)